### PR TITLE
[15.0][MIG] purchase_mrp: Migration to version 15.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT35769

Nothing to do

Depends on:

- [x] https://github.com/OCA/OpenUpgrade/pull/3744

### CODE REVIEW

**Comment by ernestotejeda** [comment]
ping @pedrobaeza

File: openupgrade_scripts/scripts/purchase_mrp/15.0.1.0/upgrade_analysis.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@
 ---XML records in module 'purchase_mrp'---
 NEW ir.model.access: purchase_mrp.access_mrp_bom_line_purchase_user
 NEW ir.model.access: purchase_mrp.access_mrp_bom_purchase_user
+# NOTHING TO DO
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/purchase_mrp/15.0.1.0/upgrade_analysis.txt]: This change is done in a copy of the file, `upgrade_analysis_work.txt`.

Comment (ernestotejeda) [reply openupgrade_scripts/scripts/purchase_mrp/15.0.1.0/upgrade_analysis.txt]: change done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**